WordPressAuthenticator/OAuth/OAuthTokenRequestBody.swift

```diff
@@ -6,16 +6,34 @@ struct OAuthTokenRequestBody: Encodable {
     let clientSecret: String
     let audience: String
     let code: String
-    let codeVerifier: String
+    let rawCodeVerifier: String
```

Comment:
I considered keeping this called `codeVerifier`, but though it be clearer to have "raw" in the name when the type is `String`.

I also considered adding a computed `codeVerifier: PKCE.CodeVerifier` property. I decided not to because the `init?(value:)` method can fail and it was either force unwrap under the assumption that the Google backend will never send back an invalid one (because it wouldn't allow us to send it an invalid one) or return an `Optional`. Moreover, there is no existing reader of that value outside of the tests... YAGNI.

Reply:
Your comment made me realize I was approaching the type in the context of `Decodable` (where it's handy to keep `String` properties to leverage the compiler-generated decoding) but this type is actually used for encoding data. 😅 

See 03f377c8.